app/models/bgs_dependents/base.rb

```diff
@@ -2,6 +2,14 @@
 
 module BGSDependents
   class Base < Common::Base
```

Comment:
Please document class attributes and methods with [YARD](https://yardoc.org/)

Reply:
Added comments for YARD documentation.